src/core/ext/filters/max_age/max_age_filter.c

```diff
@@ -384,3 +386,34 @@ const grpc_channel_filter grpc_max_age_filter = {
     grpc_call_next_get_peer,
     grpc_channel_next_get_info,
     "max_age"};
+
+static bool maybe_add_max_age_filter(grpc_exec_ctx* exec_ctx,
+                                     grpc_channel_stack_builder* builder,
+                                     void* arg) {
+  const grpc_channel_args* channel_args =
+      grpc_channel_stack_builder_get_channel_arguments(builder);
+  const grpc_arg* a =
+      grpc_channel_args_find(channel_args, GRPC_ARG_MAX_CONNECTION_AGE_MS);
+  bool enable = false;
+  if (a != NULL && a->type == GRPC_ARG_INTEGER && a->value.integer != INT_MAX) {
```

Comment:
I'm not suggesting that we have this function return something other than a bool or that we change all the callers of this function to do any special error handling.  I'm just saying that the convention we seem to have adopted for channel args in general is that if the arg is of the wrong type, then we revert to the default, and I'm not sure why this particular arg should break that convention.

And in any case, regardless of what default we want to use, I think we can still use `grpc_channel_arg_get_integer()` here.